packages/kbn-gateway/src/config/config_service.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { get } from 'lodash';
+import { firstValueFrom } from 'rxjs';
+import { fromRoot } from '@kbn/utils';
+import { RawConfigService, ConfigPath } from '@kbn/config';
+import { Type } from '@kbn/config-schema';
+import { getArgValues } from './read_argv';
+
+const CONFIG_CLI_FLAGS = ['-c', '--config'];
```

Comment:
We accept a path to the config file via `-c` or `--config` just like we do for `bin/kibana`